domManipulation.js

```diff
@@ -0,0 +1,132 @@
+import {
+  handleIntialAddTask,
+  handleIntialDeleteTask,
+  handleInputField,
+  handleDone,
+  handleDelete,
+  handleEdit,
+} from "./handler.js"
+
+import { doneIcon, editIcon, deleteIcon } from "./Helpers/icons.js"
+import { convertTime } from "./Helpers/time.js"
+
+const cardsDOM = document.querySelector("#cards")
+
+export function createCard({ itemId, title, createdAt, done, doneIn }) {
+  const card = document.createElement("div")
+  card.setAttribute("data-id", itemId)
+  card.classList.add("card")
+
+  const cardHeader = document.createElement("div")
+  cardHeader.setAttribute("data-id", itemId)
+  cardHeader.classList.add("cardHeader")
+
+  const cardFooter = document.createElement("div")
+  cardFooter.setAttribute("data-id", itemId)
+  cardFooter.classList.add("cardFooter")
+
+  const pTitle = document.createElement("p")
```

Comment:
please use meaningful variable name bhai. In this case, it could be `cardParagraphTitle`